components/service/glean/src/main/java/mozilla/components/service/glean/testing/GleanTestRule.kt

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.testing
+
+import android.content.Context
+import androidx.work.testing.WorkManagerTestInitHelper
+import mozilla.components.service.glean.Glean
+import mozilla.components.service.glean.config.Configuration
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class GleanTestRule(
+    val context: Context
+) : TestRule {
```

Comment:
FYI: There's also the [TestWatcher](https://junit.org/junit4/javadoc/4.12/org/junit/rules/TestWatcher.html) helper that may make this code a little bit easier since you only need to override the methods for the events you are about (e.g. `starting()`) - example in AC: https://github.com/mozilla-mobile/android-components/blob/e14ea8efdf52d26afabca331667a1838a7197558/samples/browser/src/androidTest/java/org/mozilla/samples/browser/rules/WebserverRule.kt#L21-L37

Reply:
Oh, great! Thanks, I didn't know about that!